### DISCUSSION TITLE
How to press Enter button for search box

### DISCUSSION THREAD

**Discussion by hayat-khan-space** [kaliiiiiiiiii/Selenium-Driverless#295] [Q&A]
from selenium_driverless.sync import webdriver
from selenium_driverless.types.by import By

options = webdriver.ChromeOptions()
with webdriver.Chrome(options=options) as driver:
    driver.get('http://www.google.com')
    search_box = driver.find_element(By.NAME, 'q', timeout=10000)
    search_box.send_keys("hi how r u song")
    search_box.click()
    # how to press Enter Button here as click is not working

**Comment by ttlns** [comment]
See https://github.com/kaliiiiiiiiii/CatRoom/blob/6c4fa9d58d42cf209d41102d664757bb61ea4528/tests/make_screenshots.py#L31
